tests/beacon/state_machines/test_proposer.py

```diff
@@ -0,0 +1,149 @@
+import pytest
+
+from eth.constants import (
+    ZERO_HASH32,
+)
+from eth.beacon.state_machines.forks.serenity import (
+    SerenityBeaconStateMachine,
+)
+from eth.beacon.db.chain import BeaconChainDB
+
+from eth.beacon.helpers import (
+    get_block_committees_info,
+    get_new_recent_block_hashes,
+)
+
+
+@pytest.fixture
+def fixture_sm_class(config):
+    return SerenityBeaconStateMachine.configure(
+        __name__='SerenityBeaconStateMachineForTesting',
+        config=config,
+    )
+
+
+@pytest.fixture
+def initial_chaindb(base_db,
+                    genesis_block,
+                    genesis_crystallized_state,
+                    genesis_active_state):
+    chaindb = BeaconChainDB(base_db)
+    chaindb.persist_block(genesis_block)
+    chaindb.persist_crystallized_state(genesis_crystallized_state)
+    chaindb.persist_active_state(genesis_active_state, genesis_crystallized_state.hash)
+    return chaindb
+
+
+@pytest.mark.parametrize(
+    (
+        'num_validators,'
+        'cycle_length,min_committee_size,shard_count'
+    ),
+    [
+        (1000, 20, 10, 100),
+    ]
+)
+def test_import_block_one(fixture_sm_class,
+                          initial_chaindb,
+                          genesis_block):
+    chaindb = initial_chaindb
+
+    # Create the first block
+    block_1_shell = genesis_block.copy(
+        parent_hash=genesis_block.hash,
+        slot_number=genesis_block.slot_number + 1,
+    )
+    sm = fixture_sm_class(chaindb, block_1_shell)
+    active_state_1 = sm.compute_per_block_transtion(
+        sm.crystallized_state,
+        sm.active_state,
+        block_1_shell,
+        sm.chaindb,
+        sm.config,
+    )
+    block_1 = block_1_shell.copy(
+        active_state_root=active_state_1.hash,
+    )
+    sm.import_block(block_1)
+
+
+@pytest.mark.parametrize(
+    (
+        'num_validators,'
+        'cycle_length,min_committee_size,shard_count'
+    ),
+    [
+        (1000, 20, 10, 100),
+    ]
+)
+def test_propose_block_and_validate_attestation(fixture_sm_class,
```

Comment:
Similar lack of assertions (though I recognize that the `validate_` calls are effectively assertions.